packages/react/src/components/progress/progress.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import { Progress, ProgressProps } from '@onfido/castor-react';
+import React, { useEffect, useState } from 'react';
+import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
+
+const size = ['regular', 'large'] as const;
+const hideLabel = [false, true] as const;
+const children = ['', 'Custom label'] as const;
+
+export default {
+  title: 'React/Progress',
+  component: Progress,
+  argTypes: {
+    children: {
+      description: 'Optional label.',
+    },
+    value: {
+      description: 'Current value',
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    min: {
+      description: 'Optional minimum value',
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    max: {
+      description: 'Optional maximum value',
+      table: {
+        defaultValue: { summary: '100' },
+      },
+    },
+    size: {
+      control: { type: 'radio', options: size },
+      table: {
+        type: { summary: size.toString() },
+        defaultValue: { summary: 'text' },
+      },
+    },
+    hideLabel: {
+      description: 'Hide label',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+  },
+  args: {
+    children: '',
+    hideLabel: false,
+    max: 100,
+    size: 'regular',
+    value: 25,
+  },
+  parameters: { display: 'flex' },
+} as Meta<ProgressProps>;
+
+export const Playground: Story<ProgressProps> = (props: ProgressProps) => (
+  <Progress {...props} />
+);
+
+export const Size = reactMatrix(Progress, { size });
+Size.argTypes = omit<ProgressProps>('size');
+
+export const CustomLabel = (props: ProgressProps) => <Progress {...props} />;
+CustomLabel.argTypes = omit<ProgressProps>('children');
+CustomLabel.args = {
+  children: 'Progress: 25%',
+};
+
+export const WithoutLabel = (props: ProgressProps) => <Progress {...props} />;
+WithoutLabel.argTypes = omit<ProgressProps>('children');
+WithoutLabel.args = {
+  children: null,
+  hideLabel: true,
+};
+
+export const AllCombinations = reactMatrix(
```

Comment:
I would suggest to make this a last story, because it is not used for humans but for automated testing.

Also, I believe CSS version is missing "All Combinations"

Reply:
Done, thanks!